### PR TITLE
Pass callback to the jQuery method

### PR DESCRIPTION
**Pull request by andizer**
## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where a deprecated jQuery function was used.

## Relevant technical choices:

* used an upgrade path recommended by jQuery to replace a deprecated jQuery method call

## Milestone

* [ ] I've attached the next release's milestone to this pull request.

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* On `trunk`
* Install the `Enable jQuery Migrate Helper` plugin made by WordPress
* Navigate to the test helper page and see an error is given
* Checkout this branch
* See that the test helper isn't given an error.
* Merge 
### Test instructions for QA when the code is in the RC

* [x] QA should use the same steps as above.

QA can test this PR by following these steps:

*

Fixes #

### CODE REVIEW

**Review by hansjovis — CHANGES_REQUESTED** [review]
CR: 🚫 

jQuery suggests fixing it differently.

File: src/admin-page.php [hunk 1]
```diff
@@ -114,7 +114,7 @@ static function( $block ) {
 	private function masonry_script() {
 		?>
 		<script type="text/javascript">
-			jQuery( window ).load( function() {
+			jQuery( function() {
```
[the remark this answers is not in the document]
Comment (andizer) [reply src/admin-page.php:117]: I just followed the implementation we've used at other places in the Yoast SEO plugin. I am fine with this 👍

**Comment by andizer** [comment]
Regression testing isn't that easy, because on older version of WordPress no errors will be given. The migrate class is causing this error. I would say for this fix the step should be enough.

**Review by increddibelly — APPROVED** [review]

**Review by hansjovis — APPROVED** [review]

**Comment by hansjovis** [comment]
With regression testing I meant that we should still test whether the functionality that it touches still works as expected.

In this case this would be that the Masonry script still loads on window load and the layout of the test helper page is still as it used to be.

**Comment by hansjovis** [comment]
Acceptance test: ✅ 

* Warning is gone.
* Layout is still as expected:
Screenshot 2020-12-01 at 09 30 20

**Pull request merged**